.gitignore

```diff
@@ -19,8 +19,8 @@ dist
 .e2e-containers.json
 coverage
 TODO
-schema.yaml
-schema.json
+/schema.yaml
+/schema.json
```

Comment:
Limited to `/` in order to be able to create the file `packages/specs/src/components/schema.yaml`.

@rijkvanzanten Do you remember why this has been added? (https://github.com/directus/directus/pull/8349/commits/fc1c87ac02d2622931defbf4b51cc5ce74042f44) Are those expected in another directory?

Reply:
It was just so I wouldn't accidentally commit my local schema snapshot debugging attempts 😁

Reply:
I see, thanks 😁 Should we leave it like this or maybe expand it to `/api/schema.yaml`?

Reply:
The `/` prefix should be fine :